src/screens/ContactScreen/elements/Header.js

```diff
@@ -88,7 +88,7 @@ const styles = StyleSheet.create({
     fontStyle: 'normal',
     fontWeight: 'bold',
     fontSize: 16,
-    color: COLORS.black,
+    color: '#000000',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value has been hardcoded to `'#000000'`. It's better to use a constant for this, as it was done before. This will make the code more maintainable and consistent.
```diff
-    color: '#000000',
+    color: colors.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->